example/user-guides/getting-started/prometheus.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prometheus
+spec:
+  serviceAccountName: prometheus
+  resources:
+    requests:
+      memory: 400Mi
```

Comment:
Is this part needed for the example?

Reply:
I agree with removing it as well, as it could be misunderstood as a recommendation.

Reply:
:+1: